### DISCUSSION TITLE
include lora in the merger

### DISCUSSION THREAD

**Discussion by DjNastyMagic** [wkpark/sd-webui-model-mixer#118] [Ideas]
I want to say that your extension, is within the top 10 best extensions to become available to automatic1111, and in stable diffusion history.
Seems like someone would have proposed this before. Maybe I'm just asking for insanity here, as I'm sht at coding, supermerger has  Lora to checkpoint for merger, is there anyway to do this within your extension? Meaning to have the ability to save the combined model to include loras. Could be save from prompt to checkpoint as an option or have fields. We can see them in the prompt but they seem to act differently when added afterwards, anyway to have them be part of the calculation that happens when we the modals are mixed?

**Comment by wkpark** [comment]
> I want to say that your extension, is within the top 10 best extensions to become available to automatic1111, and in stable diffusion history. Seems like someone would have proposed this before. Maybe I'm just asking for insanity here, as I'm sht at coding, supermerger has Lora to checkpoint for merger, is there anyway to do this within your extension? Meaning to have the ability to save the combined model to include loras. Could be save from prompt to checkpoint as an option or have fields. We can see them in the prompt but they seem to act differently when added afterwards, anyway to have them be part of the calculation that happens when we the modals are mixed?

Thank you for your compliments!

The model-merger utilises sd-webui's internal LoRA mechanism for LoRA. in fact, model-merger does not contain LoRA-specific routines to merge LoRAs.
but supermerger supports LoRA by separate/different LoRA specific routines to merge in it.

so, you can merge multiple LoRAs by input your LoRAs in the prompt -> sd-webui will merge/mix LoRAs internally -> model-mixer can utilise the results of mixed LoRAs by sd-webui optionally (Save current checkpoint - option: `with Loras`).

TL;DR
Save current chechpoint accordion menu: with `with LoRAs` option, you can merge multiple LoRAs to checkopoint

**Reply by DjNastyMagic** [reply]
awesome, thank you!!
